pkgs/kde/gear/kdenlive/default.nix

```diff
@@ -43,7 +51,7 @@ mkKdeDerivation {
     mlt
     shared-mime-info
     libv4l
-  ];
+  ] ++ lib.optional enableGlaxnimate glaxnimate;
```

Comment:
And this is still optional (again, why?)

Reply:
sorry, my mistake, fixed in https://github.com/NixOS/nixpkgs/pull/324552